flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamMockEnvironment.java

```diff
@@ -215,6 +189,31 @@ public Void answer(InvocationOnMock invocationOnMock) throws Throwable {
 		}
 	}
 
+	private <T> void addBufferToOutputList(RecordDeserializer<DeserializationDelegate<T>> recordDeserializer,
+									   NonReusingDeserializationDelegate<T> delegate, Buffer buffer,
```

Comment:
In test scope, this is not a problem, but in main scope the style guidelines (and check style plugin) do not allow leading whitespace. Personally, I don't think it's good practice to differentiate between tests and main code. You might consider removing the leading space here, too (for example one argument per line).